Ginger/GingerCoreNET/ValueExpressionLib/ValueExpression.cs

```diff
@@ -1365,12 +1367,17 @@ private void ReplaceGeneralFunctionsWithValue(string p, string[] a)
                         FuncSplit.RemoveAll(x => x.Equals(""));
                         List<string> parameters = Regex.Split(FuncSplit[0].ToString(), paramPattern).ToList<string>();
                         parameters.RemoveAll(y => y.Equals(""));
-                        parameters = parameters.Where(z => z.Contains("\"")).Select(z => z.Replace("\"", "")).ToList<string>();
-
+                        
+                        parameters[0] = parameters.FirstOrDefault().Trim('\"');
+                        
                         object[] listOfParams = new object[parameters.Count];
                         int index = 0;
                         foreach (var item in parameters)
                         {
+                            if (item.Equals(","))
```

Comment:
why to have this change?

Reply:
Earlier every function has one argument only now this function has more than one argument and all those are separated by commas.